components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -358,17 +358,26 @@ export interface ClientHeaderFields {
     clientRegion?: string;
 }
 
+const WORKSPACE_MAXIMUM_TIMEOUT_HOURS = 24;
+
 export type WorkspaceTimeoutDuration = string;
 export namespace WorkspaceTimeoutDuration {
     export function validate(duration: string): WorkspaceTimeoutDuration {
+        duration = duration.toLowerCase();
         const unit = duration.slice(-1);
         if (!["m", "h", "d"].includes(unit)) {
```

Comment:
I guess it makes sense to remove the `d` unit

Reply:
Addressed in https://github.com/gitpod-io/gitpod/pull/16267/commits/e8d85038dd7fce5a8ae6fa2bb179dc6d3898129e, thanks!